sale_commission/models/sale_commission.py

```diff
@@ -36,6 +42,70 @@ class SaleCommission(models.Model):
     sections = fields.One2many(
         comodel_name="sale.commission.section", inverse_name="commission")
     active = fields.Boolean(default=True)
+    period = fields.Selection(
```

Comment:
Why adding at commission level the settlement period? This data has sense at agent level, not at commission level